internal_transfers/actions/index.ts

```diff
@@ -1,15 +1,21 @@
-import {
-	ActionFn,
-	Context,
-	Event,
-	TransactionEvent,
-} from '@tenderly/actions';
+import { ActionFn, Context, Event, TransactionEvent } from "@tenderly/actions";
+import { partitionEventLogs } from "./src/parse";
 
 export const triggerInternalTransfersPipeline: ActionFn = async (
   context: Context,
   event: Event
 ) => {
-	const transactionEvent = event as TransactionEvent;
- 	const txHash = transactionEvent.hash;
-     console.log(`Received Settlement Event with txHash ${txHash}`);
+  const transactionEvent = event as TransactionEvent;
+  const txHash = transactionEvent.hash;
+  console.log(`Received Settlement Event with txHash ${txHash}`);
+
+  const { trades, transfers, settlements } = partitionEventLogs(
+    transactionEvent.logs
+  );
+  if (settlements.length > 1) {
+    console.error(`Two settlements in same batch ${txHash}!`);
```

Comment:
This sounds like it show fail the pipeline or can it handle this error?

Reply:
Actually I think we should be able to handle this now. Thanks for reminding me. Technically it is possible to have two settlements in the same batch so since we are aware we may as well not crash.

Reply:
Realistically though this is going to be a bad day when we see two settlements and we should probably prepare ourselves better for these events. It is possible to handle but not clear how we want to approach it.

1. I would think that solvers doing this MUST be slashed and also not rewarded for batches like this.
2. Accounting can be sorted out based on `log_index` for transfer events (treating each batch separately) -- but none of our Dune queries are ready for such a scenario (and it is doubtful that the reward mechanism for the Solver CIP outlined here 

Anyway, its not entirely clear how we should treat this scenario but it does make sense not to error here. We may even want to introduce an alert right around here

I have outlined [some possible clarifications](https://forum.cow.fi/t/cip-draft-auction-model-for-solver-rewards/1405/27?u=bh2smith) in this CIP.

Reply:
Changed Error to warning and deployed to test

* Does not crash with [warn](https://dashboard.tenderly.co/gp-v2/solver-slippage/action/4f657ddd-c054-4704-9e1e-50385ad2fc2c/execution/ad3e047c-0dc0-4a65-8e6f-aaac4935745b)